evaluators/text/BleuEvaluator/tests/test_bleuevaluator.py

```diff
@@ -22,4 +22,9 @@ def test_bleu_evaluator_caps():
     actual = 'ALL CATS ARE SO SUPER BEAUTIFUL'
     assert BleuEvaluator().evaluate(actual, desired) == 1.0
 
+def test_bleu_evaluator_fail():
+    desired = 'All cats are so super beautiful'
+    actual = 'Some dogs are also cute memes'
+    with pytest.raises(Exception):
```

Comment:
why does it raise an exception?

Reply:
Bleu is checking how similar `desired` and `actual`. Since those two are not really similar, it should fail, shouldn't it?

Reply:
no no, it should return a value smaller than one, but not raise an Exception.

Reply:
The test was checking if it was 1, otherwise, it would raise the exception, but sure, let me change it

Reply:
ah ok!!! I understand, this is not a good testing style, u should test `assert result < 1.0` or even better compute it offline and add the test there

Reply:
Oh ok, got it